test/common/page_objects/dashboard-page.js

```diff
@@ -21,7 +21,13 @@ const DashboardPage = function DashboardPage(browser) {
     return browser.visit(`/${accountId}?sort=${sort}`);
   };
 
+  this.filter = (accountId, filter) => {
+    assert(accountId, 'accountId is required');
+    return browser.visit(`/${accountId}?sort=updated&filter=${filter}`);
```

Comment:
Yes.  Its to provide a default ordering which is based on last updated time.  Keeps the ordering of the jobs in line with how the filter working for clarity when looking between the seed file.